### PR TITLE
Removing C# support for rules

### PR DESCRIPTION
**Pull request by rkhoriander**
RIP

https://docs-content-staging-pr-8944.herokuapp.com/docs/rules/guides/csharp
https://docs-content-staging-pr-8944.herokuapp.com/docs/rules/current/csharp
should both redirect to:
https://docs-content-staging-pr-8944.herokuapp.com/docs/rules

### CODE REVIEW

File: config/redirects.js [hunk 1]
```diff
@@ -2236,6 +2236,7 @@ module.exports = [
   { from: '/rules/current/redirect', to: '/rules/guides/redirect' },
   { from: '/rules/current/metadata-in-rules', to: '/rules/guides/metadata' },
   { from: '/rules/current/csharp', to: '/rules/guides/csharp' },
+  { from: '/rules/guides/csharp', to: '/rules' },
```
Comment (seejee) [inline config/redirects.js:2239]: Awesome, didn't know we had this capability! 👍

**Review by seejee — DISMISSED** [review]

File: config/redirects.js (hunk 1 above)
Comment (seejee) [reply config/redirects.js:2239]: And I assume the redirect rule above this line will chain through to this one? So `'/rules/current/csharp'` will end up at `'/rules'`?

Comment (rkhoriander) [reply config/redirects.js:2239]: Yes. Although, I guess technically I could just make it go directly to /rules without the jump.

Comment (rkhoriander) [reply config/redirects.js:2239]: In fact, that's a better choice. Fixed it.

**Review by seejee — APPROVED** [review]

**Review by tamigoodall — APPROVED** [review]

**Pull request merged**